Add more information to Performance Points Committee page, update member lists (#12070)
ppy/osu-wiki: wiki/People/Performance_Points_Committee/en.md

The **Performance Points Committee** is a panel of well-versed players and developers who facilitate and approve updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmap/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/Developers) push changes faster and with more community coordination.

The Performance Points Committee's responsibilities include:

- Reviewing and developing changes in difficulty calculation algorithms
- Organizing development efforts and helping contributors
- Deciding which changes will be deployed and used in the game

## Workflow

Whenever a new difficulty calculation proposal is created on [GitHub](https://github.com/ppy/osu), the Performance Points Committee relevant to the ruleset it's for will be tagged with a review request. After that members of the committee must review and test the proposal and write a review on its GitHub page. When there are enough committee approvals the change is considered ready for deployment and is passed to osu! developers for final code inspection before merging into the game.

> When there are enough merged proposals the osu! developers will start preparing for a performance points deployment which will recalculate all existing scores using the new algorithms. Performance Points Committees are tasked with writing a news post about the upcoming changes.

Reviewer: i would probably reword with "assisting with writing a news post" since a few times it's been other people jumping in to do it

also this is missing from the list of responsibilites which is weird, because this definitely sounds like a responsibility.

Author: yep, fixed

The **Performance Points Committee** is a panel of well-versed players and developers who facilitate and approve updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmap/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/Developers) push changes faster and with more community coordination.

The Performance Points Committee's responsibilities include:

- Organizing development efforts and helping contributors
- Reviewing and developing changes in difficulty calculation algorithms
- Deciding which changes will be deployed and used in the game
- Assisting with writing news posts for newly deployed changes

## Workflow

Whenever a new difficulty calculation proposal is created on [GitHub](https://github.com/ppy/osu), the Performance Points Committee relevant to the ruleset it's for will be tagged with a review request. After that members of the committee must review and test the proposal and write a review on its GitHub page. When there are enough committee approvals the change is considered ready for deployment and is passed to osu! developers for final code inspection before merging into the game.

> When there are enough merged proposals the osu! developers will start preparing for a performance points deployment which will recalculate all existing scores using the new algorithms. Performance Points Committees are tasked with assisting with writing a news post about the upcoming changes.

Reviewer 2: ```suggestion
The osu! developers prepare merged proposals to be deployed in batches. A deployment recalculates star ratings for all beatmaps and performance points for all scores and users. The Performance Points Committee helps write the news post that accompanies each deployment.
```
maybe? remove "enough" here again bc afaik that's not even a thing, standalone description of what a "deployment" is, some other reword

## Communication

> Performance Points Committees mainly operate on the [Performance Points Discord server](https://discord.gg/aqPCnXu) with some committees having separate servers for their respective ruleset. Committees also maintain a [Twitter account](https://x.com/pp_committee) that is being used for official announcements and difficulty calculation related news. You can also contact any of the members of the committees privately through osu! private messages.

Reviewer 2: ```suggestion
The Performance Points Committee mainly operates in the [Performance Points Discord server](https://discord.gg/aqPCnXu), with some game modes also having [separate servers](/wiki/Community/Discord_servers#community-development). The committee also maintains a [Twitter account](https://x.com/pp_committee) used for official announcements and difficulty calculation related news.
```
reword + the last part just seems unnecessary to me

## Members

> Members of the committees are hand-picked by osu! developers and/or other committee members and consist of people with a lot of experience working on the difficulty calculation system.

Reviewer 2: ```suggestion
Members of the committee are hand-picked by osu! developers and/or other committee members and consist of people with a lot of experience working on the difficulty calculation system.
```

Author: Im not sure if it should be a committee or committee_s_ since technically there are 4 of them, one for every ruleset but at the same time it probably can be said that its just one committee with subgroups

Reviewer 2: the beginning of this article made it sound like it's just one. I'm indifferent on single or plural but it should stay consistent

Author: Let's keep it single then since the article is named in single as well

### osu!

The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.

| osu! profile | GitHub profile | Join date |
| :-- | :-- | :-- |

> | ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) | [apollo-dw](https://github.com/apollo-dw) | November 9, 2021 |

Reviewer 2: use date format same as catch and mania

alternatively, YYYY-MM-DD in tables for easier columnwise comparison

| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | [stanriders](https://github.com/stanriders) | November 9, 2021 |
| ::{ flag=GB }:: [tsunyoku](https://osu.ppy.sh/users/11315329) | [tsunyoku](https://github.com/tsunyoku) | April 12, 2024 |

#### Previous members

| osu! profile | Join date | Leave date |
| :-- | :-- | :-- |
| ::{ flag=AU }:: [MBmasher](https://osu.ppy.sh/users/4498616) | November 9, 2021 | April 12, 2024 |
| ::{ flag=HU }:: [emu1337](https://osu.ppy.sh/users/2185987) | November 9, 2021 | April 12, 2024 |

### osu!taiko

> The osu!taiko panel was formed on 1 May 2022 in response to the same survey as above, with [major changes to the star rating algorithm](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit).

Reviewer 2: what "same survey above"
(just copy mania desc for that part)

#### Previous members

| osu! profile | Join date | Leave date |
| :-- | :-- | :-- |
| ::{ flag=MY }:: [vun](https://osu.ppy.sh/users/6932501) | May 1, 2022 | September 17, 2023 |

### osu!mania

The osu!mania panel was formed 22 April 2022 in response to [another community survey regarding performance points and star rating in osu!taiko and osu!mania](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, as well as in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).

| osu! profile | GitHub profile | Join date |
| :-- | :-- | :-- |
| ::{ flag=US }:: [abraker](https://osu.ppy.sh/users/4635891) | [abraker95](https://github.com/abraker95) | 22 April 2022 |
| ::{ flag=SG }:: [Evening](https://osu.ppy.sh/users/2193881) | [Eve-ning](https://github.com/Eve-ning) | 22 April 2022 |
| ::{ flag=AU }:: [molneya](https://osu.ppy.sh/users/8945180) | [molneya](https://github.com/molneya) | 22 April 2022 |
| ::{ flag=ES }:: [Quenlla](https://osu.ppy.sh/users/4725379) | [Kominaru](https://github.com/Kominaru) | 22 April 2022 |

> ### osu!catch

Reviewer 2: probably std->taiko->ctb->mania like everywhere else
